### ISSUE TITLE
need any textures?

### ISSUE THREAD

**Issue by tobyplowy** [Gerold55/Merry-Hollies#1]
@Gerold55 the title says it all

**Comment by Gerold55** [comment]
Do you wanna work on the read and unread envelopes?

**Comment by tobyplowy** [comment]
@Gerold55 eeehhhh yeh sure

16x16 is good yes?

**Comment by Gerold55** [comment]
Yeah if you need to I believe 32x32 is also fine

**Comment by Gerold55** [comment]
Grass blocks dirt blocks stone blocks need to all have a Christmas feel to them :)

**Comment by tobyplowy** [comment]
I will stick with 16x

**Comment by Gerold55** [comment]
Ok 👍

**Comment by tobyplowy** [comment]
envelope unread 
![anvelope](https://user-images.githubusercontent.com/16853304/33807441-63ce500c-ddd7-11e7-87c9-5a629034fe93.png)
envelope read 
![anvelope open](https://user-images.githubusercontent.com/16853304/33807442-63e458c0-ddd7-11e7-8398-72f44aee442e.png)

**Comment by tobyplowy** [comment]
@Gerold55 credit me as tobyplowy (aka toby109tt)

**Comment by tobyplowy** [comment]
> Grass blocks dirt blocks stone blocks need to all have a Christmas feel to them :)

yeh im going to need more info on that

**Comment by Gerold55** [comment]
Sorta like snowy maybe more of a Christmas theme to the block. Also take some artistic liberties to it. Do what you think of when you think of Christmas and try to interpret that into each different block. If that makes sense

**Comment by Gerold55** [comment]
The open and closed envelopes can they be used for the email app also?

**Comment by tobyplowy** [comment]
@Gerold55 yes that's fine (don't forget to give credit)

**Comment by Gerold55** [comment]
@tobyplowy I've done so on the readme.txt I'll add it to the forums in a few moments.

**Comment by tobyplowy** [comment]
@Gerold55 you can credit me for models as well XD
[present.zip](https://github.com/Gerold55/Merry-Hollies/files/1545962/present.zip)

**Comment by tobyplowy** [comment]
@Gerold55 the present model is made for a present block

**Comment by Gerold55** [comment]
@tobyplowy ok 👍 thanks for all of the help on this 😊

**Comment by tobyplowy** [comment]
@Gerold55 can we add oldcoder as a boss?
(He whants to destroy Christmas because he has no Soul XD )

But for real are you going to add a boss?

**Comment by Gerold55** [comment]
Yeah I plan to add a boss but idk what though. At the moment I think the basis of the games needs to be done. Like the textures Santa elves the presents a fireplace mantel and some other things. Maybe the grinch or scrooge like boss.

**Comment by Gerold55** [comment]
@tobyplowy what do you think that the grass/dirt should look like?

**Comment by tobyplowy** [comment]
> Yeah I plan to add a boss but idk what though.

What about x_mas-distructonator 2000 XD
(someting I made up for farlands but it probably won't be added)

> Like the textures Santa elves the presents a fireplace mantel and some other things.

Did someone call my name (I'm here to help)

> what do you think that the grass/dirt should look like?

I don't think you should change them they are good as is

**Comment by Gerold55** [comment]
Oh ok I want it to have a christmasy feel if you know what I mean?

**Comment by Gerold55** [comment]
@tobyplowy Do you think a Christmas themed version of the character skin be good to do?
Send me a pic of what the mob looks like if you could :)

**Comment by tobyplowy** [comment]
> Oh ok I want it to have a christmasy feel if you know what I mean?

i know what you mean XD ( ͡° ͜ʖ ͡°)

> Do you think a Christmas themed version of the character skin be good to do?

i guess

> Send me a pic of what the mob looks like if you could :)

i havent made it yet its just a idea in my head

**Comment by tobyplowy** [comment]
@Gerold55 can you also give me creddit in a txtfile included in the mod

**Comment by Gerold55** [comment]
@tobyplowy I did

**Comment by Gerold55** [comment]
Have you seen my post on the forums?

**Comment by tobyplowy** [comment]
@Gerold55 no i will have a look

**Comment by Gerold55** [comment]
I sent a message to your messages on the forum.

**Comment by tobyplowy** [comment]
@Gerold55 the present was too big so here is a [fix.zip](https://github.com/Gerold55/Merry-Hollies/files/1546223/fix.zip)

**Comment by Gerold55** [comment]
Idk what's wrong with people :(

**Comment by Gerold55** [comment]
@tobyplowy I was thinking that eventually the present could actually be a present. When a Christmas tree has been put up Santa will come and place a present underneath the tree with something random inside of it.

**Comment by Gerold55** [comment]
Someone said to use this Santa Claus but don't have permission to do so.
https://forum.minetest.net/viewtopic.php?f=9&t=4075

**Comment by tobyplowy** [comment]
> I was thinking that eventually the present could actually be a present. When a Christmas tree has been put up Santa will come and place a present underneath the tree with something random inside of it.

cool idea

**Comment by tobyplowy** [comment]
@Gerold55 i will make a santa skin

**Comment by tobyplowy** [comment]
@Gerold55 can you add a todo list of all the tings i have to do

**Comment by Gerold55** [comment]
- [ ] Update/Add Christmas themed textures 
- [ ] Santa Skin
- [ ] Elves
- [ ] Candy Canes
- [ ] Christmas Armor?
- [ ] Christmas Boss
- [ ] More Present Textures (Setup for more presents to be added)

**Comment by tobyplowy** [comment]
@Gerold55 can you get more people to help?

What about the guy that's helping you to make the laptop mod can he help maybe he can make a GUI for the present

**Comment by Gerold55** [comment]
Yeah he could and I'll look into that also I may be able to do so also :)
